includes/Sync/Product_Import.php

```diff
@@ -874,6 +1066,13 @@ protected function save_product_meta( $product_id, $data ) {
 			wp_set_object_terms( $product_id, $term_ids, 'product_cat' );
 		}
 
+		// Update custom meta.
+		if ( $data['custom_meta'] ) {
+			foreach ( $data['custom_meta'] as $meta_key => $meta_value ) {
+				update_post_meta( $product_id, $meta_key, $meta_value );
```

Comment:
Do we need any sanitization on these values before updating the meta? Or is that done upstream?